### PR TITLE
docs: update cri-related docs to revise pouch config file

### PR DESCRIPTION
**Pull request by ZouRui89**
Signed-off-by: Zou Rui <21751189@zju.edu.cn>

### Ⅰ. Describe what this PR did
This pr update the cri-related docs and script, since pouch has changed its config when starting. Adding --enable-cri=true, it allows kubernetes to work with pouch.

### CODE REVIEW

File: docs/kubernetes/pouch_with_kubernetes_deploying.md [hunk 1]
```diff
@@ -31,10 +31,25 @@ Pouch: Version 0.3.0 is recommanded.
 
 ## Install and Configure
 
+An all-in-one kubernetes cluster with frakti runtime could be deployed by running:
```
Comment (YaoZengzeng) [inline docs/kubernetes/pouch_with_kubernetes_deploying.md]: frakti/pouch

File: hack/kubernetes/allinone.sh [hunk 2]
```diff
@@ -17,17 +17,31 @@ install_pouch_ubuntu() {
...
-	sudo service pouch start
+	sudo systemctl enable pouch
+	sudo systemctl start pouch
 }
 
 install_pouch_centos() {
 	sudo yum install -y yum-utils
 	sudo yum-config-manager --add-repo http://mirrors.aliyun.com/opsx/opsx-centos7.repo
 	sudo yum update
 	sudo yum install pouch
```
Comment (zzchun) [inline hack/kubernetes/allinone.sh]: I think here should add "-y" parameter, or you will be asked to input `y` or `n`

Comment (ZouRui89) [reply hack/kubernetes/allinone.sh]: Right. :)

File: docs/kubernetes/pouch_with_kubernetes_deploying.md [hunk 3]
```diff
@@ -31,10 +31,36 @@ Pouch: Version 0.3.0 is recommanded.
...
 ### Install Pouch
 
 You can easily setup a basic Pouch environment, see [INSTALLATION.md](../../INSTALLATION.md).
 
+### Configure Pouch
+
+On Ubuntu 16.04+:
+
+```
+sed '/^ExecStart/a\ --enable-cri=true' /usr/lib/systemd/system/pouch.service
```
Comment (YaoZengzeng) [inline docs/kubernetes/pouch_with_kubernetes_deploying.md]: `sed -i` as well ?

File: docs/kubernetes/pouch_with_kubernetes_deploying.md [hunk 4]
```diff
@@ -31,10 +31,36 @@ Pouch: Version 0.3.0 is recommanded.
...
+systemctl daemon-reload
+systemctl restart pouch
+```
+
+On CentOS 7:
+
+```
+sed '/^ExecStart/a\ --enable-cri=true' /lib/systemd/system/pouch.service
```
Comment (YaoZengzeng) [inline docs/kubernetes/pouch_with_kubernetes_deploying.md]: Same as above.

**Comment by ZouRui89** [comment]
PTAL @YaoZengzeng

**Comment by YaoZengzeng** [comment]
I've tested the script on my own machine and it's excellent.

LGTM. Thanks for your work @ZouRui89 :)

**Pull request merged**